blogposts/2021/release-3.25.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Sourcegraph 3.25 release"
+publishDate: 2021-02-20T10:00-07:00
+description: "Sourcegraph 3.25 is released."
+tags: [blog, release]
+slug: "release/3.25"
+published: true
+changelogItems:
+- description: "Go `1.15` introduced changes to SSL/TLS connection validation which requires certificates to include a `SAN`. This field was not included in older certificates and clients relied on the `CN` field. You might see an error like `x509: certificate relies on legacy Common Name field`. We recommend that customers using Sourcegraph with an external database that is connected using SSL/TLS check whether the certificate is up to date. AWS RDS customers please reference [AWS' documentation on updating the SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html) for steps to rotate your certificate."
```

Comment:
@content-hacker - this is an important updated to include, but it doesn't align well with the short descriptions we normally use. Thoughts on having this be the last update, but keeping the length?

Reply:
@abschu looking at the preview, the longer length isn't jarring. If it's an important update, I would keep it at the top to make sure people see it.

I think last release @Joelkw figured out that we can't externally link from this post, it breaks it. When I look at the preview, the link doesn't work and categories are missing, likely because of the link to aws.

Reply:
Yep, one "problem" of this format is that you can't link in the description, but we do let people link in the changelog (which feeds this description).